nservicebus/upgrades/all-versions.include.md

```diff
@@ -608,7 +608,7 @@ _This package has been split into NServiceBus.DataBus.AzureBlobStorage and NServ
 | Version   | Released       | Supported until   | Notes                             |
 |:---------:|:--------------:|:-----------------:|:---------------------------------:|
 | 3.0.x     | 2018-06-07     | -                 | -                                 |
-| 2.1.x     | 2018-05-23     | 2019-06-07        | Superseded by 3.0.x               |
+| 2.1.x     | 2018-05-23     | 2020-05-29        | Extended by NServiceBus 6.4.x     |
```

Comment:
FTR, this changed because NServiceBus.Metrics.ServiceControl 2.1.0 has no dependency on NServiceBus, but 2.1.1 has. The last time the script was run, only 2.1.0 was released.